### PR TITLE
ON-11235 # Removed delay to open hint tooltip

### PR DESCRIPTION
**Pull request by mymattcarroll**

### CODE REVIEW

File: src/components/FormElementLabelContainer.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ function FormElementLabelContainer({
           {element.label}
         </label>
         {element.hint && (
-          <Tooltip title={element.hint} arrow>
+          <Tooltip title={element.hint} arrow enterTouchDelay={0}>
```
Comment (Zaxist) [inline src/components/FormElementLabelContainer.tsx:31]: Want to change the other tooltip(s ?) as well? Think theres one in the `BaseForm` component

Comment (mymattcarroll) [reply src/components/FormElementLabelContainer.tsx:31]: Agreed

Comment (mymattcarroll) [reply src/components/FormElementLabelContainer.tsx:31]: Actually, disagreed. It is on a button and a press on that should not show the tooltip, it should just execute the action. IMO

Comment (Zaxist) [reply src/components/FormElementLabelContainer.tsx:31]: Sounds good

**Pull request merged**